pkg/cmd/server/api/types.go

```diff
@@ -934,3 +940,22 @@ type LDAPQuery struct {
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string
 }
+
+type AdmissionPluginConfig struct {
+	// Location is the path to a configuration file that contains the plugin's
+	// configuration
+	Location string
+
+	// Configuration is an embedded configuration object to be used as the plugin's
+	// configuration. If present, it will be used instead of the path to the configuration file.
+	Configuration runtime.EmbeddedObject
+}
+
+type AdmissionConfig struct {
+	// PluginConfig allows specifying a configuration file per admission control plugin
+	PluginConfig map[string]AdmissionPluginConfig
```

Comment:
Fine for now, given that we use maps elsewhere, this is fine.  I think we might cut a v2 external version to allow an in-place patch.  @liggitt something to consider for eliminating our `sed` yaml mutation.